services/xkms/xkms-x509-handlers/src/test/java/org/apache/cxf/xkms/x509/validator/TrustedAuthorityValidatorCRLTest.java

```diff
@@ -91,7 +91,7 @@ public void testIsCertChainValid() throws CertificateException {
                           validator.isCertificateChainValid(Arrays.asList(certificateRoot)));
         Assert.assertTrue("wss40rev should not be valid",
                           !validator.isCertificateChainValid(Arrays.asList(certificateWss40Rev)));
-        Assert.assertTrue("wss40 should be valid",
+        Assert.assertTrue("wss40 should not be valid",
```

Comment:
@sanskruti1098  Please explain this change

Reply:
May I ask which JDK for PPCLE64 are you building with? 
IIRC RedHat openjdk 17 had a build issue with x509 test case... Temurin on PPC64LE builds clean.